### PR TITLE
docs: make nav and layout work for mobile

### PR DESCRIPTION
**Pull request by abutterworth**
Nav collapses for small screens:
image

### CODE REVIEW

File: www/src/pages/components/collapsible.mdx [hunk 1]
```diff
@@ -215,13 +215,17 @@ the component will be updated accordingly.
...
 <PropsTable {...props.data.collapsible} />
+<PropsTable {...props.data.collapsibleAdvanced} />
 <PropsTable {...props.data.collapsibleTrigger} />
 <PropsTable {...props.data.collapsibleBody} />
 <PropsTable {...props.data.collapsibleVisible} />
 
 export const query = graphql`
   query {
-    collapsible: componentMetadata(displayName: { eq: "CardGrid" }) {
+    collapsible: componentMetadata(displayName: { eq: "Collapsible" }) {
```
Comment (abutterworth) [inline www/src/pages/components/collapsible.mdx:225]: Unrelated fix to get the proper component props metadata

**Review by raq929 — APPROVED** [review]
OMG So much more functional. 🎉 
You're good to go as soon as the one broken deploy preview is working. :)
